packages/node/src/app/analytics-node.ts

```diff
@@ -7,327 +7,298 @@ import {
   CorePlugin,
   EventFactory,
   EventQueue,
-  dispatch,
-  resolvePageArguments,
-  PageParams,
-  Integrations,
+  dispatchAndEmit,
   CoreOptions,
   Callback,
   CoreSegmentEvent,
   bindAll,
+  PriorityQueue,
 } from '@segment/analytics-core'
+import { AnalyticsNodeSettings, validateSettings } from './settings'
+import { analyticsNode, AnalyticsNodePluginSettings } from './plugin'
 
 import { version } from '../../package.json'
+import { EmittedError } from './errors'
 
-/** create a derived class since we may want to add node specific things to Context later  */
+// create a derived class since we may want to add node specific things to Context later
 export class NodeContext extends CoreContext {}
 
-export type NodeSegmentEventOptions = CoreOptions & Identity
-
-export type Identity =
+/**
+ * An ID associated with the user. Note: at least one of userId or anonymousId must be included.
+ **/
+type IdentityOptions =
   | { userId: string; anonymousId?: string }
   | { userId?: string; anonymousId: string }
 
+/** Events from CoreOptions */
+export interface NodeSegmentEventOptions {
+  context?: NodeContext
+  timestamp?: CoreOptions['timestamp']
+  anonymousId?: CoreOptions['anonymousId']
+  userId?: CoreOptions['userId']
```

Comment:
It looks like these are in the top-level object passed to our `track et al` calls. Can we omit these in node so they don't also show up under options?